swan-lake/integration-tutorials/service-orchestration.md

```diff
@@ -108,90 +108,89 @@ Follow the instructions given in this section to develop the service.
     $ bal new service-orchestration
     ```
 
-2. Introduce the source code in files with the `.bal` extension (e.g., the `main.bal` file). 
-
-    Import the 
-    - `ballerina/http` module to develop the REST API and define the clients that can be used to send requests to the backend services
-    - `ballerina/log` module to log debug/error information for each client request
+2. Remove the generated content in the `main.bal` file and open the diagram view in VS Code. 
+   
+   ![Open diagram view](/learn/images/integration-tutorials/service-orchestration/open_diagram_view.gif)
 
-    ```ballerina
-    import ballerina/http;
-    import ballerina/log;
-    ```
+3. Generate record types corresponding to the payloads from the hospital and payment backend services by providing samples of the expected JSON payloads.
 
-3. Define three [configurable variables](https://ballerina.io/learn/by-example/#configurability) for the port the listener should listen on and the URLs of the backend services.
+    The payload from the hospital backend service will be a JSON object similar to the following.
```

Comment:
Need to extract out the request payload -> record generation. Right now it's under response payload -> record generation?